runners/wikiConsole/templates/createFile.asciidoc

```diff
@@ -0,0 +1,30 @@
+== Create the file <%= fileName; %>
+
+==== Prerequisites
+First, you need to open a command prompt in your current workspace. For Windows, you can use PowerShell and on Linux, you can use the Terminal.
+
+The folder where you want to create the file has to be existing. If the folder doesn’t exist you have to create it.
```

Comment:
I think 'has to exist' or 'You need to create the base folder, if it doesn't exist already. ', would be better